doc/releases/changelog-dev.md

```diff
@@ -205,4 +208,4 @@ Soran Jahangiri,
 Korbinian Kottmann,
```

Comment:
Could optionally add yourself here (this might also happen as we update the changelog)